multibranch/src/test/java/org/jenkinsci/plugins/workflow/multibranch/SCMBinderTest.java

```diff
@@ -144,171 +104,114 @@ private static void assertRevisionAction(WorkflowRun build) {
     }
 
     @Test public void exactRevisionSubversion() throws Exception {
-        story.addStep(new Statement() {
```

Comment:
So there's a pile o' diffs like this - are they largely just indent changes or should I be reading more closely?